### PR TITLE
[12.0][ADD] web_register_login

### PR DESCRIPTION
**Pull request by jarroyomorales**
@etobella

### CODE REVIEW

File: web_register_login/security/res_users_access_log_security.xml [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+        <field name="model_id" ref="web_register_login.model_res_users_access_log"/>
+        <field name="perm_create" eval="True"/>
+        <field name="domain_force">[(1, '=', 1)]</field>
+        <field name="groups" eval="[(6, 0, [ref('base.group_user')])]"/>
+    </record>
+
+    <record id="model_res_users_access_log_read_own" model="ir.rule">
+        <field name="name">Access Log Everyone can read its own</field>
+        <field name="model_id" ref="web_register_login.model_res_users_access_log"/>
+        <field name="perm_read" eval="True"/>
```
Comment (etobella) [inline web_register_login/security/res_users_access_log_security.xml:13]: Missing perm_create = True

File: web_register_login/security/res_users_access_log_security.xml [hunk 2]
```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <record id="group_admin_access_log" model="res.groups">
+        <field name="name">View Access Log</field>
+        <field name="users" eval="[(4, ref('base.group_system'))]"/>
+    </record>
+
+    <record id="model_res_users_access_log_create_rule" model="ir.rule">
```
Comment (etobella) [inline web_register_login/security/res_users_access_log_security.xml]: No sense, I can only create my own

File: web_register_login/security/res_users_access_log_security.xml [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+        <field name="perm_read" eval="True"/>
+        <field name="domain_force">[('create_uid', '=', user.id)]</field>
+        <field name="groups" eval="[(6, 0, [ref('base.group_user')])]"/>
+    </record>
+
+    <record id="model_res_users_access_log_admin_rule" model="ir.rule">
+        <field name="name">group_admin_access_log can do everything</field>
+        <field name="model_id" ref="web_register_login.model_res_users_access_log"/>
+        <field name="perm_read" eval="True"/>
+        <field name="perm_write" eval="True"/>
```
Comment (etobella) [inline web_register_login/security/res_users_access_log_security.xml]: no write, only read

File: web_register_login/models/res_users.py [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class ResUsers(models.Model):
+
+    _inherit = "res.users"
+
+    dont_register_login = fields.Boolean()
```
Comment (etobella) [inline web_register_login/models/res_users.py:11]: `dont_register_login` or `no_register_login`?

**Review by etobella — APPROVED** [review]
Only some comments

File: web_register_login/models/res_users.py (hunk 4 above)
Comment (jarroyomorales) [reply web_register_login/models/res_users.py:11]: Both LGTM, we can keep it like `dont_register_login` to make it easier to read...

**Pull request merged**